### PR TITLE
Feature/login page

### PR DESCRIPTION
**Pull request by khal45**
Finished creating the UI template for the login page

### CODE REVIEW

**Comment by khal45** [comment]
The page looks decent enough

File: views/login.html
Comment (Bobjayafam) [inline views/login.html]: You have a typo here Id should be username

File: public/styles.css
Comment (Bobjayafam) [inline public/styles.css]: Why do you have -parent in the class names? I don’t think you should have them there. Imagine when you have to use JavaScript to get by class name and then do .parent on it. It may confuse you

**Review by Bobjayafam — APPROVED** [review]
Good job! I left a few comments. Attend to them. Otherwise everything looks good. But I will approve so that after you make the changes then you can merge and continue working.

**Pull request merged**
